onelogin_aws_cli/__init__.py

```diff
@@ -6,39 +6,13 @@
 import configparser
 import getpass
 import os
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ElementTree
 
 import boto3
 from onelogin.api.client import OneLoginClient
 
 CONFIG_FILENAME = ".onelogin-aws.config"
-
```

Comment:
When trying to build a test case, I found that keeping this function in the `__init__.py` file was causing circular dependencies with imports, so I moved it into its own module , `onelogin_aws_cli.userquery`